cli/commands/hclfmt/action_test.go

```diff
@@ -74,6 +75,21 @@ func TestHCLFmt(t *testing.T) {
 
 			assert.Equal(t, original, actual)
 		})
+
+		// Finally, check to make sure the file in the `.history` folder was ignored and untouched
+		t.Run("history", func(t *testing.T) {
+			t.Parallel()
+
+			originalTgHclPath := "../../../test/fixtures/hclfmt/ignored/.history/terragrunt.hcl"
```

Comment:
I would recommend keeping the fixture for the unit test in the same package, e.g. `cli/commands/hclfmt/testdata/fixtures`, since we use the `test/` folder only for integration tests.

Reply:
Yes, please move `ignored/.terragrunt-cache` as well. Thanks!